src/main/java/ca/corefacility/bioinformatics/irida/ria/web/components/datatables/DataTablesColumnDefinitions.java

```diff
@@ -61,28 +67,42 @@ static DataTablesColumnDefinitions createColumnDefinition(Integer index, HttpSer
 	 * Column Name
 	 *
 	 * @return {@link String}
-	 * @see <a href="https://datatables.net/reference/option/columns.name">columns.name</a>
+	 * @see <a href=
+	 *      "https://datatables.net/reference/option/columns.name">columns.name</a>
 	 */
 	public String getColumnName() {
 		return name;
 	}
 
+	/**
+	 * Column Data
+	 * 
+	 * @return {@link String}
+	 * @see <a href=
+	 *      "https://datatables.net/reference/option/columns.data">columns.data</a>
+	 */
+	public String getData() {
```

Comment:
Added in method for `data` variable that was reporting unused compiler warnings.